src/DotLiquid/StandardFilters.cs

```diff
@@ -218,14 +234,21 @@ public static string TruncateWords(string input, int words = 15, string truncate
 
         /// <summary>
         /// Split input string into an array of substrings separated by given pattern.
+        ///
+        /// If the pattern is empty the input string is converted to an array of 1-char

```

Comment:
Add this in a `remarks` XML element after the summary element

Reply:
Done.